parm/fv3lam_rrfs.xml

```diff
@@ -1923,6 +1985,54 @@
            <scale>5.0</scale>
        </param>
 
+       <param>
+           <shortname>CRITICAL_ANGLE</shortname>
+           <pname>CANGLE</pname>
```

Comment:
@AndrewBenjamin-NOAA This field was not generated in your test. You may remove it.